src/centralserver/admin-ui/src/views/AppLogin.vue

```diff
@@ -219,18 +216,15 @@ export default (
               });
```

Comment:
This error handler starting from line 200 is handling rejected promise of `fetchUserData()`  i.e. the reject handler in` then(`